sdk/kusto/Azure.ResourceManager.Kusto/CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Release History
 
+## 1.6.0 (2025-01-16)
```

Comment:
Hi, I got this error for this date: 
"Build log #L44

Invalid date [ 2025-01-16 ]. The date for the changelog being released must be the latest in the file."

Can you help me with this?

Reply:
@guyhirPublic You need to remove `## 1.6.0-beta.1 (Unreleased)` below

Reply:
Should I keep the content or remove it all? @archerzz

Reply:
> Should I keep the content or remove it all? @archerzz

Since `1.6.0-beta.1` is never released, you need to check the contents and merge them into `1.6.0` if necessary.